web/src/main/java/co/mobileaction/example/web/controller/AdminController.java

```diff
@@ -21,12 +23,21 @@
 public class AdminController
 {
     private final IPostQueueService queueService;
-
+    private final PostService postService;

```

Comment:
we can use interface instead of implementation class
```suggestion
    private final IPostService postService;

```